btr-web/btr-main-app/interfaces/btr-bods/btr-bods-person-i.ts

```diff
@@ -37,4 +37,11 @@ export interface BtrBodsPersonI {
   missingInfoReason?: string
 
   uuid: string
+
+  sharesInConcert?: string[],
```

Comment:
maybe we can meet on this, but doesn't it make sense for this data to be inside the BodsInterestI ? Having it here makes it seem unrelated to the control information we track inside the interest, but really it is part of that information